packages/owl-bot/test/handlers.ts

```diff
@@ -188,6 +224,24 @@ class FakeConfigStore implements ConfigsStore {
   constructor(configs?: Map<string, Configs>) {
     this.configs = configs ?? new Map<string, Configs>();
   }
+  findPullRequestForChangedFiles(
+    repo: string,
+    googleapisGenCommitHash: string
+  ): Promise<string | undefined> {
+    throw new Error('Method not implemented.');
+  }
+  recordPullRequestForChangedFiles(
+    repo: string,
+    googleapisGenCommitHash: string,
+    pullRequestId: string
+  ): Promise<string> {
+    throw new Error('Method not implemented.');
```

Comment:
will there be a helper that actually creates the PR with the change set?

Reply:
The code to create PRs should be implemented outside ConfigsStore.